solarized-definitions.el

```diff
@@ -316,25 +317,30 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
              (org-hide ((t (,@fg-base03))))
              (org-todo ((t (,@fmt-bold ,@fg-base03 ,@bg-red))))
              (org-done ((t (,@fmt-bold ,@fg-green))))
-             (org-todo-kwd-face ((t (,@fg-red ,@bg-base03))))
-             (org-done-kwd-face ((t (,@fg-green ,@bg-base03))))
+             (org-todo-kwd-face ((t (,@fg-red ,@bg-base03 :box (:line-width 1 :color ,red :style released-button)))))
+             (org-done-kwd-face ((t (,@fg-green ,@bg-base03 :box (:line-width 1 :color ,green :style released-button)))))
              (org-project-kwd-face ((t (,@fg-violet ,@bg-base03))))
-             (org-waiting-kwd-face ((t (,@fg-orange ,@bg-base03))))
-             (org-someday-kwd-face ((t (,@fg-blue ,@bg-base03))))
-             (org-started-kwd-face ((t (,@fg-yellow ,@bg-base03))))
-             (org-cancelled-kwd-face ((t (,@fg-green ,@bg-base03))))
-             (org-delegated-kwd-face ((t (,@fg-cyan ,@bg-base03))))
+             (org-someday-kwd-face ((t (,@fg-magenta ,@bg-base03 :box (:line-width 1 :color ,magenta :style released-button)))))
+             (org-appt-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
+             (org-waiting-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
+             (org-started-kwd-face ((t (,@fg-yellow ,@bg-base03 :box (:line-width 1 :color ,yellow :style released-button)))))
+             (org-next-kwd-face ((t (,@fg-base03 ,@bg-yellow))))
+             (org-cancelled-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
+             (org-delegated-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
+             (org-deferred-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
```

Comment:
These faces are probably good, but they’re not in the vein of the original Vim Solarized theme.

I’m in the process of applying something like #191 to make it easier to extend Solarized, so that you could add these faces locally by doing something like
```emacs-lisp
(deftheme huyz-solarized "Extensions to the Solarized theme.")
(apply #'custom-theme-set-faces 'huyz-solarized
       (solarized-with-color-variables
        `((org-todo-kwd-face ((t (,@fg-red ,@bg-base03 :box (:line-width 1 :color ,red :style released-button)))))
           (org-done-kwd-face ((t (,@fg-green ,@bg-base03 :box (:line-width 1 :color ,green :style released-button)))))
           (org-someday-kwd-face ((t (,@fg-magenta ,@bg-base03 :box (:line-width 1 :color ,magenta :style released-button)))))
           (org-appt-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
           (org-waiting-kwd-face ((t (,@fg-orange ,@bg-base03 :box (:line-width 1 :color ,orange :style released-button)))))
           (org-started-kwd-face ((t (,@fg-yellow ,@bg-base03 :box (:line-width 1 :color ,yellow :style released-button)))))
           (org-next-kwd-face ((t (,@fg-base03 ,@bg-yellow))))
           (org-cancelled-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
           (org-delegated-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))
           (org-deferred-kwd-face ((t (,@fg-cyan ,@bg-base03 :box (:line-width 1 :color ,cyan :style released-button)))))))

(customize-set-variables
 '(custom-enabled-themes (huyz-solarized solarized))
```

I actually do something similar locally, because I don’t always agree with Solarized, and I plan to add some variants to the repo as well, but I want _this_ theme to match the original as much as possible.